controllers/drplacementcontrol_controller.go

```diff
@@ -303,8 +304,8 @@ func DRClusterUpdateOfInterest(oldDRCluster, newDRCluster *rmn.DRCluster) bool {
 		}
 	}
 
-	// Exhausted all failover activation checks, this update is NOT of interest
-	return false
+	// Exhausted all failover activation checks, the only interesting update is deleting a drcluster.
+	return drClusterIsDeleted(newDRCluster)
```

Comment:
I agree with Shyam. You are better off returning true on line 269 and allowing all drpcs to reconcile.

Reply:
I started this change by returning true in line 269, but unfortunately this does not work.

Deleting creates an *update* event, when the new object has a deletionTimeStamp. When the object is actually removed from the system, we get a *delete* event.

In our case this event is not relevant. It will happen when the drpolicy is deleted the the ramen removes the finalizers from the the drcluster.

Reply:
I see, so adding the `deletionTimeStamp` ends up firing for `Updated`. Got it.
So how about if in line 307, you return true every time the newObject (newDRCluster) has a non-zero deletionTimeStamp? 
I think avoiding adding the new code (`DRPCsUsingDRCluster` and `DRPCsUsingDRPolicy`) is desirable.

Reply:
I did not try the change without modifying FilterDRCluster - but if we don't modify it we will trigger a reconcile only in the DRPCs which are failing over to the deleted cluster, which is always no DRPC.

We an simplify by returning all DPPCS but if we have to add code it makes sense to add the right code which is only few more lines.